Packs/qualys/Scripts/QualysCreateIncidentFromReport/QualysCreateIncidentFromReport.yml

```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: QualysCreateIncidentFromReport
+  version: -1
+name: QualysCreateIncidentFromReport
+script: ''
+type: python
+subtype: python2
```

Comment:
```suggestion
subtype: python3
```

Reply:
this isnt a new script, I just converted from yaml. Its python 2